features/admin.organizations.v1/pages/organizations.tsx

```diff
@@ -573,6 +573,7 @@ const OrganizationsPage: FunctionComponent<OrganizationsPageInterface> = (
                             defaultSearchOperator="co"
                             triggerClearQuery={ triggerClearQuery }
                             data-componentid={ `${ testId }-list-advanced-search` }
+                            data-testid={ `${ testId }-list-advanced-search` }
```

Comment:
Why do we need to add `data-testid` ? Can't we use `data-componentid` for writing E2E tests ?

Reply:
Currently, the [AdvancedSearchWithBasicFiltersPropsInterface](https://github.com/wso2/identity-apps/blob/master/features/admin.core.v1/components/advanced-search-with-basic-filters.tsx#L39) extends the deprecated **TestableComponentInterface** instead of the **IdentifiableComponentInterface**. As a result, the **AdvancedSearchWithBasicFilters** component only accepts the **[data-testid](https://github.com/wso2/identity-apps/blob/master/features/admin.core.v1/components/advanced-search-with-basic-filters.tsx#L187)** prop, which is used as a prefix for all its [sub-components](https://github.com/wso2/identity-apps/blob/master/features/admin.core.v1/components/advanced-search-with-basic-filters.tsx#L388). [Other pages ](https://github.com/wso2/identity-apps/blob/master/features/admin.applications.v1/pages/applications.tsx#L677)currently use the data-testid prop.

We have two options:

1. Pass the **data-testid** prop in the same way as other pages.
2. Update the **AdvancedSearchWithBasicFilters** component to extend the **IdentifiableComponentInterface**, and modify all instances where **data-testid** is passed to pass **data-componentid**.

Reply:
Since we are moving away from testablecomponentinterface, the right fix here, IMO, would be to update the usage of TestableComponentInterface to IdentifiableComponentInterface, by following the below steps.

- Remove the newly added `data-testid`
- Search and replace `TestableComponentInterface` with `IdentifiableComponentInterface`, and `testId` with `componentId`, in advanced-search-with-basic-filters.tsx.
- update data-testid to data-componentid in https://github.com/wso2/identity-apps/blob/master/features/admin.core.v1/components/advanced-search-with-basic-filters.tsx#L187

that's it!

You wouldn't need to touch the data-testid passed to any of the children components in advanced-search-with-basic-filters.tsx, as that'd be out of scope of the PR.

Reply:
Let's go with the current changes, as data-testid has been passed in other places as well.